apps/frontend/src/news/__tests__/News.test.tsx

```diff
@@ -1,14 +1,14 @@
 import nock from 'nock';
 import { render, waitFor } from '@testing-library/react';
 import { MemoryRouter, Route } from 'react-router-dom';
-import { NewsOrEventResponse } from '@asap-hub/model';
+import { NewsResponse } from '@asap-hub/model';
 import { authTestUtils } from '@asap-hub/react-components';
 import { news } from '@asap-hub/routing';
 
-import NewsOrEvent from '../NewsOrEvent';
+import NewsOrEvent from '../News';
 import { API_BASE_URL } from '../../config';
 
-const newsOrEvent: NewsOrEventResponse = {
+const newsOrEvent: NewsResponse = {
```

Comment:
again - should be just `news`

Reply:
Irritatingly NewsOrEvent and NewsAndEvent exist. I have intentionally left this alone